packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -50,4 +50,21 @@ export default class AnthosTenantsService {
       },
     );
   }
+
+  getStorageVolumes(serviceName) {
+    return this.$http
+      .get(`/dedicated/anthos/tenants/${serviceName}/storage/netapp/svms`, {
+        headers: {
+          'x-pagination-mode': 'CachedObjectList-Pages',
```

Comment:
Any reason to prefer using lowercase? (For consistency we mainly use `X-Pagination-Mode`)

Reply:
No reason actually. Its changed as suggested